server/utils/matchUtils.ts

```diff
@@ -145,6 +150,48 @@ export const confirmationSummaryCardRows = (
   ]
 }
 
+export const spaceBookingSummaryCardRows = (
+  premisesName: string,
+  apType: ApType,
+  dates: PlacementDates,
+  gender: Gender,
+  essentialCharacteristics: Array<PlacementCriteria>,
+  desirableCharacteristics: Array<PlacementCriteria>,
+): Array<SummaryListItem> => {
+  return [
+    summaryCardTextRow('Approved Premises', premisesName),
```

Comment:
We've tended to use miniature utils for creating Summary Card rows in the past as it easy to scan and test functions like this. I'm not opposed to the approach you've gone for here but I think we should be consistent and either go for having *all* rows as `summaryCardTextRow`s or the other approach